### ISSUE TITLE
init_app override constructor parameters

### ISSUE THREAD

**Issue by nicbon** [jfinkels/flask-restless#389]
The following code:
```python
manager = APIManager(flask_sqlalchemy_db=db)
manager.init_app(app)
```
set the session property of APIManager to None producing surprising AttributeError while querying the apis. I think it would be more intuitive to not override previously set attributes.

**Comment by jfinkels** [comment]
This is probably related to issue #313. The way I have written `init_app` is incorrect, it needs to be redone.

**Comment by jfinkels** [comment]
Merged a fix for this in pull request #392.
